uber/configspec.ini

```diff
@@ -783,6 +788,67 @@ by_mail_us_only = boolean(default=True)
 # "social_media" list.
 [social_media_placeholders]
 
+[cherrypy]
+checker.check_skipped_app_config = boolean(default=False)
+
+engine.autoreload.on = boolean(default=False)
+
+# True to enable (or False to disable) the Sideboard profiler.
```

Comment:
Remove reference to "Sideboard" in these comment blocks (there are several).

Reply:
I actually removed the sideboard profiler since it wasn't really doing anything cherrypy's current profiler wasn't. I just pulled these config settings.